include/eve/arch/riscv/rvv_common_masks.hpp

```diff
@@ -0,0 +1,56 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/function/friends.hpp>
```

Comment:
That's not an include you should use

Reply:
Ok, I wanted to use it as I need `self_neq`, but it seems I do not need this. Removed.